designsafe/settings/common_settings.py

```diff
@@ -499,6 +499,10 @@
 TAS_CLIENT_SECRET = os.environ.get('TAS_CLIENT_SECRET', None)
 TAS_URL = os.environ.get('TAS_URL', None)
 
+# Allocations
+#
+ALLOCATIONS_TO_EXCLUDE = ["DesignSafe-DCV"]
```

Comment:
Should this be a setting derived from `designsafe.dev`, so updating wouldn't require a code change?

Reply:
Good Point, made the adjustments.